python/sqlcommenter-python/google/cloud/sqlcommenter/fastapi.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
```

Comment:
Although SQLCommenter is donated to OpenTelemetry, but the migration is going to take time. For now, should we add the copyright as added in all other existing files in this repo's python packages ?